frontend/components/password-forgot/pug/password-forgot.pug

```diff
@@ -15,13 +15,10 @@
 			.md-toolbar-tools
 				h2 Forgot Password
 		.userContent(layout="column")
-			p Please enter your username and email
+			p Please enter your username or email
 			md-input-container.md-block
-				label Username
-				input(type='text', ng-model='vm.username', ng-required, autofocus)
-			md-input-container.md-block
-				label Email
-				input(type='text', ng-model='vm.email', ng-required, ng-keypress="vm.requestPasswordChange($event)")
+				label Username or Email
+				input(type='text', ng-model='vm.usernameOrEmail', ng-required, ng-keypress="vm.requestPasswordChange($event)", autofocus)
```

Comment:
Can we also have a "confirm password" field here? Similar to the sign up page.

Reply:
I have added the confirm password fields to the password reset.
I have also tested to see whether, the user can submit the password if passwords do no match. 
The user can only submit if both password's are correct.